django/test/utils.py

```diff
@@ -836,6 +836,9 @@ def disable(self):
 def tag(*tags):
     """Decorator to add tags to a test class or method."""
     def decorator(obj):
-        setattr(obj, 'tags', set(tags))
+        if hasattr(obj, 'tags'):
+            obj.tags = obj.tags | set(tags)
```

Comment:
Maybe `obj.tags.update(tags)`?

Reply:
@sir-sigurd this would update the parent tags as well.

Reply:
Indeed, then `obj.tags = obj.tags.union(tags)`, it's a bit faster.